app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
```

Comment:
Zaimplementowali Panowie tworzenie userów samemu, ale to nie daje jeszcze tworzenia sesji, możliwości korzystania ze zmiennej `current_user`, helpera `authenticate_user!` itd. Polecam wykorzystać gem devise do zarządzania userami, będzie dużo łatwiej.